### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30026]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Angela Fowler (Accessibility)
Josh Kim (Accessibility)

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
Currently, the modal has the following issues:

- The first heading is a `h3`, which jumps two heading levels upon page load and triggers an axe error
- The list of dependents' names are `h3`s which would skip a heading level if the first issue is fixed

### Link, screenshot or steps to recreate
<img width="1783" alt="Screen Shot 2022-05-09 at 4 39 50 PM" src="https://user-images.githubusercontent.com/14154792/167495699-d5f248bf-c84d-4b94-9ef4-402e176f6c6a.png">

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
We recommend making the first heading a `h1` and the list of dependents` names as `h2`s. A `h1` is appropriate in this instance as:
- the modal appears on page load
- no other `h1` is available when the modal is present
- when open, the modal's contents is the page itself and the other `h1` should be unreachable to screen readers

### Acceptance criteria:

- [ ] The first heading is a `h1`
- [ ] The list of dependents' names are `h2`s

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by joshkimux** [comment]
@Mottie updated this ticket!

**Comment by tblackwe** [comment]
This issue still exists in the codebase. we will accept it into the backlog and work on it.

**Comment by Mottie** [comment]
This Dependency Verification MODAL isn't being worked on any more. We opted to get the va-notify team to email the Veteran instead of depending on them to visit this app.
